app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,47 @@
+class ArticlesController < ApplicationController
+	
+	http_basic_authenticate_with name: "rushi", password: "secret", except: [:index, :show]
```

Comment:
secret 😱

Reply:
I was checking out some feature. thats not necessary.